### PR TITLE
fix data fetching + split fetch and recon

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

**Comment by agramfort** [comment]
good to go from my end

**Review by larsoner — APPROVED** [review]

File: scripts/processing/Makefile [hunk 1]
```diff
@@ -11,8 +11,13 @@ help:
 check:
 	$(PYTHON) check_system.py
 
-fetch-recon:
+clean:
+	rm -rf ../../ds119
```
Comment (jasmainak) [inline scripts/processing/Makefile]: ds119? did you see the `clean` directive in this PR: https://github.com/mne-tools/mne-biomag-group-demo/pull/25/files?

File: scripts/processing/Makefile [hunk 2]
```diff
@@ -69,5 +74,6 @@ profile:
 	mprof plot -o group_average_source
 
 all:
-	$(MAKE) fetch-recon
+	$(MAKE) fetch
+	$(MAKE) recon
```
Comment (jasmainak) [inline scripts/processing/Makefile:76]: need to update the help commands too, but I can do it. Just let me know about the `make clean`

**Comment by agramfort** [comment]
ok comments addressed. Merging

**Comment by jasmainak** [comment]
Thanks!

**Pull request merged**